subjects/java/checkpoints/area-calculator/README.md

```diff
@@ -0,0 +1,82 @@
+## Area Calculator
+
+### Instructions
+
+Create a class `AreaCalculator` that has an overloaded method `calculate` to calculate the area of different shapes. The method should be able to calculate the area for:
+
+- A circle (given the radius)
+- A rectangle (given the width and height)
+- A triangle (given the base and height)
+- A square (given the side length)
+
+#### Formulas for area calculations:
+
+- Circle: Area = 𝜋 × radius²
+
+- Rectangle: Area = width × height
+
+- Triangle: Area = ½ × base × height
+
+- Square: Area = side²
+
+### Expected Class
+
+```java
```

Comment:
Are the learners supposed to use (or allowed to use) the `Math` package/library?

Reply:
no need for them to use the Math library but yeah they are allowed to use it.

Reply:
I am asking since the value of `pi` might be something you either know or not. Just to be sure I would add the value to use or the fact that they can use the `Math.PI` (provided by `java.lang.Math`)

Reply:
yeah i got what you meant, so they are allowed to use it no problem